app/controllers/admin/merchants_controller.rb

```diff
@@ -17,23 +17,35 @@ def update
         redirect_to admin_merchant_path(@merchant)
     end
   end
-  
+
   def show
     @merchant = Merchant.find(params[:id])
   end
-  
+
   def edit
     @merchant = Merchant.find(params[:id])
   end
-  
+
+  def new
+    @merchant = Merchant.new
+  end
+
+  def create
+    merchant = Merchant.new(name: params[:merchant][:name], enabled: false)
+    if merchant.save
+      flash[:notice] = 'New merchant has been created!'
```

Comment:
You can disable the merchant here instead with `merchant.update(enabled: false)`

Reply:
@redferret ohhh okay I'll give that a try and then re-push!

Reply:
@redferret  went ahead and did that - and it looks much better!